.github/workflows/deploy-prep.yml

```
@@ -0,0 +1,739 @@
+          fi
+          
+          echo "image-tag=${TAG}" >> $GITHUB_OUTPUT
+          echo "Using image tag: ${TAG}"
+
+      - name: Validate deployment readiness
+        id: validation
+        env:
+          ENVIRONMENT: ${{ steps.env-decision.outputs.environment }}
+        run: |
+          DEPLOY_READY="true"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)